src/reflection/FStar.Reflection.NBEEmbeddings.fs

```diff
@@ -673,22 +733,6 @@ let e_exp =
 
 let e_binder_view = e_tuple2 e_bv (e_tuple2 e_aqualv (e_list e_term))
 
-let e_attribute  = e_term
```

Comment:
Do we need to remove these? I am guessing these functions are not used currently but still any harm in keeping them?

Reply:
These aren’t removed, they're moved a little earlier in the module (l.625).